.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: python
+python:
+  - "3.6"
+node_js:
+  - "8"
+cache:
+  pip: true
+addons:
+  apt:
+    packages:
+    - httpie
+install:
+  - set -eo pipefail
+  - pip install flake8
```

Comment:
you could also do pip install httpie here instead of doing it as part of apt.

Reply:
that's cleaner, I like that.